source/vocab/iiif-presentation-annotations.ttl

```diff
@@ -0,0 +1,118 @@
+@prefix iiif_pres3: <http://iiif.io/api/presentation/3#> . # Funkar denna urin?
+@prefix oa: <http://www.w3.org/ns/oa#> . # Funkar denna urin?
+@prefix rdfs: <http://www.w3.org/2000/01/rdf-schema#> .
+@prefix owl: <http://www.w3.org/2002/07/owl#> .
+@prefix xsd: <http://www.w3.org/2001/XMLSchema#> .
+@prefix skos: <http://www.w3.org/2004/02/skos/core#> .
+@prefix prov: <http://www.w3.org/ns/prov#> .
+@prefix sdo: <http://schema.org/> .
+@prefix dc: <http://purl.org/dc/terms/> .
+@prefix bf2: <http://id.loc.gov/ontologies/bibframe/> .
+@prefix relsubtype: <http://id.loc.gov/vocabulary/preservation/relationshipSubType/> .
+@prefix : <https://id.kb.se/vocab/> .
+
+##
+# IIIF PRESENTATION API 3.0
+
+:id owl:equivalentProperty iiif_pres3:id .
+:type owl:equivalentProperty iiif_pres3:type .
```

Comment:
These are _probably_ not defined in their (non-RDF-documented)  vocabulary. I gather you've taken them from [their specific JSON-LD-context](https://iiif.io/api/presentation/3/context.json)? They are only defined there as aliases of  `@id` and `@type`. We (as in XL) do not use such aliases in the default context that _we_ associate with KBV (and use throughout in XL). Of course anyone using KBV from _another_ system-specific context are free to use any particular forms of JSON they need (ideally with motivations), as long as we keep the contexts separate (though one can build on another).

In any case, they should not be added as terms to KBV (nor as aliases to the XL default context at id.kb.se).